api/source/service/AssetService.js

```diff
@@ -1181,7 +1181,7 @@ exports.xccdfFromAssetStig = async function (assetId, benchmarkId, revisionStr =
       revisionStrResolved = `V${result[0].version}R${result[0].release}`
     }
     else {
-      let revParse = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+      let revParse = dbUtils.extractRevisionDetails(revisionStr)
       revId = `${benchmarkId}-${revParse[1]}-${revParse[2]}`
```

Comment:
There are several instances where this structure is used too, and these will need to change to the 0,1 items in the array as well.